### ISSUE TITLE
"Google refine" is damaged

### ISSUE THREAD

**Issue by ianks** [CSCI-4830-002-2014/challenge-week-4#25]
Getting a message like this?
![damaged](https://www.dropbox.com/s/2rhnl9i4xqt9r51/Screenshot%202014-09-21%2018.51.11.png?dl=1)

Mac OSX 10.9.5.

Going to try the unstable [2.6.beta.1](https://github.com/OpenRefine/OpenRefine/releases/tag/2.6-beta.1) version. Anyone else run into this issue?

**Comment by JoshFerge** [comment]
You have to go into Settings > Security and Privacy then click "Anywhere" under "Allow Apps to be Downloaded From:"

**Comment by wannabeCitizen** [comment]
Look at an early comment I made to Chris Wittenberg. This problem is due to your Mac having strict privacy settings. Read back to that issue where I linked how to fix this.

-Mike

**Comment by ianks** [comment]
I do not think this was a privacy issue. Typically they notify of that, eh? I downloaded the `2.6.beta.1` build and it doesn't throw the same error. So far so good.

**Comment by ianks** [comment]
Ah, I stand corrected. Got it working with the app permissions thing.

Can't tell if I should blame Apple or Google... decisions.
